Fix declaration file errors in 4.4.3

Trying out version 4.4.3 and getting the following errors:

```
> npx tsc
node_modules/deno.ns/dist/deno/stable/functions/sleepSync.d.ts:2:38 - error TS2724: 'Deno' has no exported member named 'sleepSync'. Did you mean 'seekSync'?

2 export declare const sleepSync: Deno.sleepSync;
                                       ~~~~~~~~~

node_modules/deno.ns/dist/deno/stable/functions/test.d.ts:2:29 - error TS2307: Cannot find module '@fromdeno/test' or its corresponding type declarations.

2 export { test, tests } from "@fromdeno/test";
                              ~~~~~~~~~~~~~~~~

node_modules/deno.ns/dist/deno/stable/variables/ppid.d.ts:2:33 - error TS2724: 'Deno' has no exported member named 'ppid'. Did you mean 'pid'?

2 export declare const ppid: Deno.ppid;
                                  ~~~~


Found 3 errors.
```

After adding `@fromdeno/test`:

```
> npx tsc
node_modules/deno.ns/dist/deno/stable/functions/sleepSync.d.ts:2:38 - error TS2724: 'Deno' has no exported member named 'sleepSync'. Did you mean 'seekSync'?

2 export declare const sleepSync: Deno.sleepSync;
                                       ~~~~~~~~~

node_modules/deno.ns/dist/deno/stable/variables/ppid.d.ts:2:33 - error TS2724: 'Deno' has no exported member named 'ppid'. Did you mean 'pid'?

2 export declare const ppid: Deno.ppid;
                                  ~~~~


Found 2 errors.
```

--- a/src/deno/stable/functions.ts
+++ b/src/deno/stable/functions.ts
@@ -58,7 +58,6 @@ export { removeSync } from "./functions/removeSync.js";
 export { rename } from "./functions/rename.js";
 export { renameSync } from "./functions/renameSync.js";
 export { shutdown } from "./functions/shutdown.js";
-export { sleepSync } from "./functions/sleepSync.js";
 export { stat } from "./functions/stat.js";
 export { statSync } from "./functions/statSync.js";
 export { symlink } from "./functions/symlink.js";

--- a/src/deno/stable/functions/sleepSync.ts
+++ b/src/deno/stable/functions/sleepSync.ts
@@ -1,6 +1,7 @@
 ///<reference path="../lib.deno.d.ts" />
 
-// @ts-expect-error sleepSync is documented as unstable but available in stable
+// This file is not exported in functions.ts; we must come back when sleepSync is stable
+// @ts-expect-error sleepSync was previously documented as unstable but available in stable
 export const sleepSync: Deno.sleepSync = function sleepSync(millis: number) {
   // buffer will never change to awaited value,
   // Atomics will block the thread for given milliseconds and return

--- a/src/deno/stable/variables/ppid.ts
+++ b/src/deno/stable/variables/ppid.ts
@@ -1,4 +1,3 @@
 ///<reference path="../lib.deno.d.ts" />
 
-// @ts-expect-error ppid is documented as unstable but available in stable
-export const ppid: Deno.ppid = process.ppid;
+export const ppid: typeof Deno.ppid = process.ppid;

